packages/fuels/src/lib.rs

```diff
@@ -72,6 +72,7 @@ pub mod prelude {
             codec::{LogDecoder, LogId, LogResult},
             traits::Signer,
         },
+        macros::setup_program_test,
```

Comment:
`std` feature doesn't imply `fuels-test-helpers` so having it here isn't a guarantee that it will have all it needs to work (i.e. what is gated behind `fuels-test-helpers`).

I suggest a `cargo new` with `fuels` as a dep with `default-dependencies=false` then try and see if a setup program test macro will fail to compile under various feature combos.

Reply:
Ah, I'm sorry it actually does imply the test helpers :S 

saw
```
  "fuels-test-helpers?/std",
```
and went.. ok that's a problem

but above it we have
```
  "dep:fuels-test-helpers",
```

which guarantees the dep. 

Weird way to write it but :D